k8s/operator/helm/templates/00_olm.yaml

```diff
@@ -124,10 +124,10 @@ spec:
           command:
           - /bin/catalog
           args:
-          - '-namespace'
+          - '--namespace'
           - {{ .Values.olmNamespace }}
-          - -configmapServerImage={{ if .Values.registry }}{{ .Values.registry }}/quay.io-operator-framework-{{ else }}quay.io/operator-framework/{{ end }}configmap-operator-registry:latest
-          - -util-image
+          - --configmapServerImage={{ if .Values.registry }}{{ .Values.registry }}/quay.io-operator-framework-{{ else }}quay.io/operator-framework/{{ end }}configmap-operator-registry:latest
+          - --util-image
           -  {{ if .Values.registry }}{{ .Values.registry }}/quay.io-operator-framework-olm{{ else }}quay.io/operator-framework/olm@sha256:b706ee6583c4c3cf8059d44234c8a4505804adcc742bcddb3d1e2f6eff3d6519{{ end }}
           image: {{ if .Values.registry }}{{ .Values.registry }}/quay.io-operator-framework-olm{{ else }}quay.io/operator-framework/olm@sha256:b706ee6583c4c3cf8059d44234c8a4505804adcc742bcddb3d1e2f6eff3d6519{{ end }}
```

Comment:
Looks like you are using a custom registry and this issue is happening because the image isn't pinned to the same SHA hash in the custom registry.
Instead of changing the flags, can you just add the SHA to the custom registry template here so that we are always consistently using the same images, and so that this template stays inline with the original olm yaml.

Reply:
Thanks, got it.❤️

Reply:
I meant that you could update this PR to also update it for the Pixie charts!
Lmk if you would like to make a PR, else I can fix this on our end soon.